### ISSUE TITLE
SatelliteImage warns every time it is instantiated.

### ISSUE THREAD

**Issue by erikmannerfelt** [GlacioHack/GeoUtils#211]
**Describe the bug**
Every time a `SatelliteImage` is instantiated, a warning is triggered: `Parse metadata from file not implemented`

**To Reproduce**
```python
>>> import geoutils as gu
>>> gu.SatelliteImage(gu.datasets.get_path("landsat_B4"), silent=True)
/home/erik/.local/miniforge3/lib/python3.8/site-packages/geoutils/satimg.py:355: UserWarning: Parse metadata from file not implemented. None
  warnings.warn(f"Parse metadata from file not implemented. {fn_meta}")
```

**Expected behavior**
The warning should not be triggered every time a `SatelliteImage` is made. The easiest fix is to default the `read_from_meta` argument to `False` (it is currently `True`, thus calling the function that has not been implemented). The more long-term fix is to either implement `SatelliteImage.__parse_metadata_from_file` or remove it.

Additionally, I think the warning should be replaced with a `NotImplementedError`.

**System (please complete the following information):**
 - OS: Linux
 - Environment `geoutils=0.0.2` from conda-forge

**Additional context**
Note: I am actually the one that made the warning come up: In #201, I changed it to warn as `flake8` complained about an empty function.

**Comment by adehecq** [comment]
Is this not solved now?

**Comment by adehecq** [comment]
By #214.

**Comment by rhugonnet** [comment]
Yes! But this one is "hidden"!
Solved by #222
